cmd/entrypoint/post_writer.go

```diff
@@ -20,3 +20,41 @@ func (*realPostWriter) Write(file string) {
 		log.Fatalf("Creating %q: %v", file, err)
 	}
 }
+
```

Comment:
Two more functions are introduced (1) to create a file and write the specified content in the file (2) to create a directory and a symbolic link.